Added the ability to switch between basic authentication and token-based authentication (#2867)
zowe/zowe-explorer-vscode: packages/zowe-explorer/CHANGELOG.md

- Added a confirmation dialog when the encoding is changed for an unsaved data set or USS file. [#2911](https://github.com/zowe/zowe-explorer-vscode/pull/2911)
- Added a "Copy Relative Path" context option for USS files and directories in the tree view. [#2908](https://github.com/zowe/zowe-explorer-vscode/pull/2908)
- Added a "Copy Name" context option for data sets, jobs and spool files in the tree view. [#2908](https://github.com/zowe/zowe-explorer-vscode/pull/2908)

> - To switch from Basic authentication to Token based authentication and viceversa. [#2867](https://github.com/zowe/zowe-explorer-vscode/pull/2867)

Reviewer: We might want to adjust the wording a bit here to make it clear on what has changed:

```suggestion
- Added the ability to switch between basic authentication and token-based authentication. [#2867](https://github.com/zowe/zowe-explorer-vscode/pull/2867)
```
